frontend/src/main.tsx

```diff
@@ -2,9 +2,12 @@ import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
 import './index.css'
 import App from './App.tsx'
+import {BrowserRouter as Router} from "react-router-dom";
 
 createRoot(document.getElementById('root')!).render(
-  <StrictMode>
-    <App />
-  </StrictMode>,
+    <StrictMode>
+        <Router>
```

Comment:
You usually won't render a Router directly. Instead, you'll render a router that is more specific to your environment such as a BrowserRouter in web browsers :-)